tests/sav/nitpick_args1.res

```diff
@@ -1,4 +1,4 @@
-../lib/standard/stream.nit:426,6--17: Documentation warning: Undocumented property `buffer_reset`
+../lib/standard/stream.nit:451,6--17: Documentation warning: Undocumented property `buffer_reset`
```

Comment:
Can't you just document the property and improve the global happyness in the world?